tasks/sep/metal-001-MCP-L1/input.json

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
I'm not able to recreate this input.json file. It's likely my environment isn't configured properly. 

```
go run op-chain-ops/cmd/op-upgrade/main.go --l1-rpc-url https://eth-sepolia.g.alchemy.com/v2/5ief20x2sn8ULZYkzYm2UtVeE3M1en9y --chain-ids 1740 --superchain-target sepolia --outfile input.json --deploy-config ./packages/contracts-bedrock/deploy-config
```

This is the command I'm using. 

I've got the monorepo `develop` branch checked out and the latest `main` of the superchain registry. 

I've edited go.mod to point to my local superchain registery. 

![Screenshot 2024-04-15 at 17 12 18](https://github.com/ethereum-optimism/superchain-ops/assets/15608778/587db33a-14cd-439f-8d00-c0db4d860adb)


I am getting a warning message, which could be something to do with it!

Reply:
Can you try with on [this PR](https://github.com/ethereum-optimism/optimism/pull/10160#pullrequestreview-2002197177) of the monorepo? Between the time I originally generated it and now, a PR was merged to the monorepo which changed the bindings, so now a call fails, giving the execution reverted error that you see

Reply:
![Screenshot 2024-04-15 at 17 48 15](https://github.com/ethereum-optimism/superchain-ops/assets/15608778/4d076059-f867-4773-8139-dabba9dcd2bc)

New error: 

```
blaine@blaines-MacBook-Pro optimism % go run op-chain-ops/cmd/op-upgrade/main.go --l1-rpc-url https://eth-sepolia.g.alchemy.com/v2/5ief20x2sn8ULZYkzYm2UtVeE3M1en9y --chain-ids 1740 --superchain-target sepolia --outfile input.json --deploy-config ./packages/contracts-bedrock/deploy-config
WARN [04-15|17:48:02.970] Cannot find deploy config for network, so validity checks will be skipped name="Metal Sepolia" deploy-config-name= path=./packages/contracts-bedrock/deploy-config err="deploy config at packages/contracts-bedrock/deploy-config/.json not found: open packages/contracts-bedrock/deploy-config/.json: no such file or directory"
INFO [04-15|17:48:03.261] Metal Sepolia                            l1-chain-id=11,155,111 l2-chain-id=1740
INFO [04-15|17:48:03.261] Detecting on chain contracts
INFO [04-15|17:48:03.721] L1CrossDomainMessenger                   version=1.4.1 address=0x5d335aa7d93102110879e3b54985c5f08146091e
INFO [04-15|17:48:03.721] L1ERC721Bridge                           version=1.1.2 address=0x5d6ce6917dbeeacf010c96bffdabe89e33a30309
INFO [04-15|17:48:03.721] L1StandardBridge                         version=1.1.1 address=0x21530aadf4dcfb9c477171400e40d4ef615868be
INFO [04-15|17:48:03.721] L2OutputOracle                           version=1.3.1 address=0x75a6b961c8da942ee03ca641b09c322549f6fa98
INFO [04-15|17:48:03.721] OptimismMintableERC20Factory             version=1.1.1 address=0x49ff2c4be882298e8ca7decd195c207c42b45f66
INFO [04-15|17:48:03.721] OptimismPortal                           version=1.7.2 address=0x01d4dfc994878682811b2980653d03e589f093cb
INFO [04-15|17:48:03.721] SystemConfig                             version=1.3.1 address=0x5d63a8dc2737ce771aa4a6510d063b6ba2c4f6f2
INFO [04-15|17:48:03.721] Upgrading to the following versions
INFO [04-15|17:48:03.721] L1CrossDomainMessenger                   version=v2.3.0 address=0xd3494713a5cfad3f5359379dfa074e2ac8c6fd65
INFO [04-15|17:48:03.721] L1ERC721Bridge                           version=v2.1.0 address=0xae2af01232a6c4a4d3012c5ec5b1b35059caf10d
INFO [04-15|17:48:03.721] L1StandardBridge                         version=v2.1.0 address=0x64b5a5ed26dcb17370ff4d33a8d503f0fbd06cff
INFO [04-15|17:48:03.721] L2OutputOracle                           version=v1.8.0 address=0xf243bed163251380e78068d317ae10f26042b292
INFO [04-15|17:48:03.721] OptimismMintableERC20Factory             version=v1.9.0 address=0xe01efbeb1089d1d1db9c6c8b135c934c0734c846
INFO [04-15|17:48:03.721] OptimismPortal                           version=v2.5.0 address=0x2d778797049fe9259d947d1ed8e5442226dfb589
INFO [04-15|17:48:03.721] SystemConfig                             version=v1.12.0 address=0xba2492e52f45651b60b8b38d4ea5e2390c64ffb1
CRIT [04-15|17:48:05.894] error op-upgrade                         err="upgrading SystemConfig: strconv.ParseUint: parsing \"\": invalid syntax"
exit status 1
```

Reply:
Hashes **match** of `input.json`

Mine (monorepo branch: `mds/registry-upgrade-and-abi-fix` - _to get it to match I had to manually remove a new line at the end of this file_): 
```bash
# Command: 
SYSTEM_CONFIG_START_BLOCK=5304055 go run op-chain-ops/cmd/op-upgrade/main.go --l1-rpc-url https://eth-sepolia.g.alchemy.com/v2/5ief20x2sn8ULZYkzYm2UtVeE3M1en9y --chain-ids 1740 --superchain-target sepolia --outfile input.json --deploy-config ./packages/contracts-bedrock/deploy-config


openssl dgst -sha512 input.json | awk '{print $2}'
4328f91a4baf953ff7c07e47f5058175a0b889b2ca3e986727ea9ef56a88ed975deaf6d7e3145f388c740c09d433bacb1b89fa685d43e31044e96339fb66e426
```


PR (this branch):
```bash
# Command: 
SYSTEM_CONFIG_START_BLOCK=5304055 go run op-chain-ops/cmd/op-upgrade/main.go --l1-rpc-url https://eth-sepolia.g.alchemy.com/v2/5ief20x2sn8ULZYkzYm2UtVeE3M1en9y --chain-ids 1740 --superchain-target sepolia --outfile input.json --deploy-config ./packages/contracts-bedrock/deploy-config

openssl dgst -sha512 input.json | awk '{print $2}'
4328f91a4baf953ff7c07e47f5058175a0b889b2ca3e986727ea9ef56a88ed975deaf6d7e3145f388c740c09d433bacb1b89fa685d43e31044e96339fb66e426
```

Reply:
Awesome, glad you got it to match! Will resolve this as a result :)